Sources/Sentry/SentryDispatchSourceWrapper.m

```diff
@@ -0,0 +1,32 @@
+#import "SentryDispatchSourceWrapper.h"
+#import "SentryDispatchQueueWrapper.h"
+
+@implementation SentryDispatchSourceWrapper {
+    SentryDispatchQueueWrapper *_queueWrapper;
+    dispatch_source_t _source;
+}
+
+- (instancetype)initTimerWithInterval:(uint64_t)interval
+                               leeway:(uint64_t)leeway
+                                queue:(SentryDispatchQueueWrapper *)queueWrapper
+                         eventHandler:(void (^)(void))eventHandler
+{
+    if (!(self = [super init])) {
```

Comment:
I'll make the change but it's very annoying to have to change around code to satisfy a tool like this. IMO it'll actually be worse, because without the early return the failure path is now implicit. The same failure modes are still present, but the fact that the tooling treats it differently because of the way the code is written is not great :/